docs/docs/15-concepts.md

```diff
@@ -48,7 +53,10 @@ Kargo seeks to progress from one stage to another.
 
 ### What is a Warehouse?
 
-A **warehouse** is a _source_ of freight. A warehouse subscribes to one or more:
+A **warehouse** is a _source_ of freight. It is where a collection of 'Freight'
+gets stored before the 'Freight' is deployed anywhere.
+
+A warehouse subscribes to one or more:
```

Comment:
Even in the real word, I think warehouses are a somewhat fuzzy concept. Is a warehouse a sort of archive where physical things are stored, possibly long-term? Or is it more of a fulfillment/shipping hub? It can be one or the other, or both.

Kargo warehouses are really about fulfillment. I'm not sure it would be accurate to say they _store_ freight. Freight are instances of a CRD. So, really, _Kubernetes_ stores your Freight. The first few sections of this page stay away from technical details. CRDs are not mentioned until later. But I think the statement that warehouses _store_ freight may set the reader up for confusion when they do read about the CRD later.